PCSX2/pcsx2
# Misc: More DuckStation backports

### Description of Changes

 - FullscreenUI: Fix gaps between some UI elements 
 - FullscreenUI: Fix minimal output latency toggle
 - FullscreenUI: Add horizontal padding to menu windows 
 - FullscreenUI: Use native file selector on Flatpak
 - Qt: Add option to pause when controller is disconnected
 - 3rdparty/cubeb: Sync to 19fcbef 
 - 3rdparty/cubeb: Only include connected devices when enumerating wasapi 
 - GS/Vulkan: Simplify loader using DynamicLibrary 
 - SettingsInterface: Fix TinyString helper 
 - Host: Add plural translation support 
 - Error: Strip trailing whitespace from Windows errors 
 - MTGS: Eliminate redundant SynchronousMTGS checks in Release build
 - Config: Remove redundant GS setting macros 
 - GS: Make VSync a boolean toggle 
 - InputManager: Add push to toggle for macros 
 - AudioStream: Vectorize volume application
 - FullscreenUI: Warning fix
 - USB-Eyetoy: Fix unused comprLen warning 
 - VMManager: Rewrite input profile hotkey priority logic
 - CI/AppImage: Don't bundle libssl/libcrypto

### Rationale behind Changes

See commit list.
Closes #11242.
Closes #11237.
Closes #11230.

### Suggested Testing Steps

 - Check Big Picture UI.
 - Test pause on controller disconnect (you'll need to actually have it bound to a virtual PS2 controller).
 - Check sound output.
 - Make sure Vulkan renderer still works.
 - Make sure VSync still works.
 - Test new push-to-toggle macro feature.
 - Check input profiles.
 - Make sure AppImage still runs.
